### PR TITLE
remove bad pkg names when finding deps

### PR DESCRIPTION
**Pull request by schloerke**
Issue: https://github.com/rstudio/renv/issues/228

PR task list:
- [NA] Update NEWS
- [NA] Add tests (if possible)
- [NA] Update documentation with `devtools::document()`

### CODE REVIEW

File: R/install_tutorial_dependencies.R [hunk 1]
```diff
@@ -2,6 +2,9 @@ get_needed_pkgs <- function(dir) {
 
   pkgs <- unique(renv::dependencies(dir, quiet = TRUE)$Package)
 
+  # remove packages with name "cannot open connection" (or any _pkg_ with a space in its name)
+  pkgs <- pkgs[!grepl(" ", pkgs)]
```
Comment (kevinushey) [inline R/install_tutorial_dependencies.R]: Just to be safe: is this an issue with `renv`'s behavior, or is it done to handle code that might look like `library(<your package here>)`; e.g. with placeholder code?

Comment (schloerke) [reply R/install_tutorial_dependencies.R]: Ha!  Making an issue now.

It's unexpected behavior for `renv`.  I'll link here soon.

Comment (kevinushey) [reply R/install_tutorial_dependencies.R]: Thank you :-)

Comment (schloerke) [reply R/install_tutorial_dependencies.R]: https://github.com/rstudio/renv/issues/228

**Pull request merged**